src/scss/components/toast/_index.scss

```diff
@@ -2,8 +2,10 @@
 
 hx-toast {
   background-color: $gray-0;
+  background-color: var(--hxToast-bgcolor, $gray-0);
```

Comment:
Please update the file to use our new design tokens.  Use the closet available token.